sdks/java/io/parquet/src/main/java/org/apache/beam/sdk/io/parquet/ParquetIO.java

```diff
@@ -1097,6 +1102,12 @@ public Sink withConfiguration(Configuration configuration) {
       return toBuilder().setConfiguration(new SerializableConfiguration(configuration)).build();
     }
 
+    /** Specify row-group size; if not set, a default will be used by the underlying writer. */
+    public Sink withRowGroupSize(int rowGroupSize) {
+      checkArgument(rowGroupSize > 0, "rowGroupSize should be positive");
```

Comment:
nit: "**must** be"

Reply:
Done; also added a note about the zero default value in the comment.